src/dmd/backend/gother.c

```diff
@@ -1488,6 +1488,7 @@ STATIC void accumda(elem *n,vec_t DEAD, vec_t POSS)
                 vec_free(Pr); vec_free(Dr);
                 break;
 
+            case OPrelconst:
```

Comment:
This is close, but not quite right. The fix is to add a case for `OPbt`. I'll work on that tomorrow.